docs/configuring_and_running.md

```diff
@@ -239,6 +258,148 @@ module = "/path/to/bar.wasm"
 entrypoint = "goodbye  # Executes the `goodbye()` function in the module (instead of `_start`)
 ```
 
+## Using a Bindle Instead of a `modules.toml`
```

Comment:
Would it make sense to also add an example of using Bindle with a `modules.toml` config file?

Reply:
I think we are going to deprecate that use case.